Tests/DatadogIntegrationTests/Scenarios/RUM/RUMCommonAsserts.swift

```diff
@@ -60,4 +60,12 @@ extension RUMSessionMatcher {
 
         return sessionMatchers.first
     }
+
+    class func assertViewWasInitiallyActive(_ viewVisit: ViewVisit) {
+        XCTAssertTrue(try XCTUnwrap(viewVisit.viewEvents.first?.view.isActive))
+    }
+
+    class func assertViewWasEventuallyInactive(_ viewVisit: ViewVisit) {
+        XCTAssertFalse(try XCTUnwrap(viewVisit.viewEvents.last?.view.isActive))
+    }
```

Comment:
I like this assertions 👍. But instead of repeating them in all tests, how about moving it to the `RUMSessionMatcher.init()'s` implementation? We already raise few `RUMSessionConsistencyException` for common inconsistencies if spotted in RUM Session. This would keep integration tests cleaner 👌.

Reply:
That's interesting. I can see how this would help have 0 changes in the integration tests files. Let's try this out.

Reply:
As a bonus, we will validate it also in some unit tests as we use it there as well 😉. `RUMSessionMatcher` is a fairly new construct, so it doesn't have 100% adoption in unit tests yet.